### ISSUE TITLE
These files won't open in my slicer. Says they are corrupt.

### ISSUE THREAD

**Issue by Kyleboedeker1** [keebio/kbo-5000-case#3]

**Comment by nooges** [comment]
Which slicer are you using?

**Comment by Kyleboedeker1** [comment]
Creality 4.8

**Comment by Kyleboedeker1** [comment]
I think I figured it out. For some reason what downloaded when I right clicked and selected save as was corrupted every time. But if I just click the download button, I successfully ended up with an archive of two step files.

**Comment by nooges** [comment]
Yeah, you need to click on Raw and save that file, otherwise you are saving a webpage.
